backend/src/app.js

```diff
@@ -24,6 +24,7 @@ import unitRoute from './routes/unitRoute.js';
 import attributeRoute from './routes/attributeRoute.js';
 import couponRoute from './routes/couponRoute.js';
 import adminRoute from './routes/adminRoute.js';
+import roleRoute from './routes/adminRoute.js';
```

Comment:
```suggestion
import roleRoute from './routes/roleRoute.js';
```

Te dejo la corrección para que le hagas commit. Siempre ojo con los copypaste en las importaciones.